test/expected.txt

```diff
@@ -300,6 +300,20 @@
 || Enter command: || [Command entered:  exit]
 || Exiting Address Book as requested ...
 || ===================================================
+|| List of all tags added/deleted during this session:
+|| 	+ Betsy Choo [secretive]
+|| 	+ Charlie Dickson [school]
+|| 	+ Charlie Dickson [friends]
```

Comment:
excellent! i am very happy to see you have followed the instructions perfectly :+1: